<?xml version="1.0"?>
<launch>

	<node pkg="move_base" type="move_base" respawn="true"
		name="move_base" output="screen">
		<rosparam
			file="$(find dalu_robot)/config/navigation/costmap_common_params.yaml"
			command="load" ns="global_costmap" />
		<rosparam
			file="$(find dalu_robot)/config/navigation/costmap_common_params.yaml"
			command="load" ns="local_costmap" />
		<rosparam
			file="$(find dalu_robot)/config/navigation/local_costmap_params.yaml"
			command="load" />
		<rosparam
			file="$(find dalu_robot)/config/navigation/global_costmap_params.yaml"
			command="load" />
		<rosparam
			file="$(find dalu_robot)/config/navigation/base_local_planner_params.yaml"
			command="load" />
		<remap from="/cmd_vel" to="/cmd_vel_intermediate" />
		<remap from="/odom" to="/wheel_odom" />
	</node>

</launch>
